src/components/RelicFilterBar.jsx

```diff
@@ -237,51 +240,51 @@ export default function RelicFilterBar(props) {
     <Flex vertical gap={2}>
       <Flex gap={10}>
         <Flex vertical flex={1}>
-          <HeaderText>Part</HeaderText>
+          <HeaderText>{t('relicFilterBar.part')}</HeaderText>
           <FilterRow name='part' tags={partsData} flexBasis='15%'/>
         </Flex>
         <Flex vertical style={{ height: '100%' }} flex={1}>
-          <HeaderText>Enhance</HeaderText>
+          <HeaderText>{t('relicFilterBar.enhance')}</HeaderText>
           <FilterRow name='enhance' tags={enhanceData} flexBasis='15%'/>
         </Flex>
         <Flex vertical flex={0.5}>
-          <HeaderText>Grade</HeaderText>
+          <HeaderText>{t('relicFilterBar.grade')}</HeaderText>
           <FilterRow name='grade' tags={gradeData} flexBasis='15%'/>
         </Flex>
         <Flex vertical flex={0.25}>
-          <HeaderText>Verified</HeaderText>
+          <HeaderText>{t('relicFilterBar.verified')}</HeaderText>
           <FilterRow name='verified' tags={verifiedData} flexBasis='15%'/>
         </Flex>
         <Flex vertical flex={0.25}>
-          <HeaderText>Equipped</HeaderText>
+          <HeaderText>{t('relicFilterBar.equipped')}</HeaderText>
           <FilterRow name='equippedBy' tags={equippedByData} flexBasis='15%'/>
         </Flex>
         <Flex vertical flex={0.4}>
-          <HeaderText>Clear</HeaderText>
+          <HeaderText>{t('clear')}</HeaderText>
           <Button icon={<ClearOutlined/>} onClick={clearClicked} style={{ flexGrow: 1, height: '100%' }}>
-            Clear all filters
+            {t('relicFilterBar.clearbutton')}
```

Comment:
There's some inconsistent case naming going on here. I think we have to decide on one of the following and stick with it for consistency

* RelicFilterBar.ClearButton
* relicFilterBar.clearButton
* relicfilterbase.clearbutton

I have a slight preference for the first one

Reply:
yeah I was initially wanting to keep everything lower case with uppercase being reserved for i18n specific things (e.g. "keyWithCount" and "key") but that's proven to not really hold up so well, especially with the keys for stats. will probably move to the first option.